CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- (Breaking) Commands are divided to subcommands: most of the existing command are inside `stream` subcommand, and `generate` command inside `mock-data` subcommand
```

Comment:
:+1:

Reply:
To make it absolutely crystal clear, we could add (at sub-list level):
- `streamr generate`  is now `streamr mock-data generate`
- `streamr subscribe`  is now `streamr stream subscribe`
- `streamr publish`  is now `streamr stream publish`
- `streamr list`  is now ` streamr stream list`
- `streamr show` is now `streamr stream show`
- `streamr create`  is now `streamr stream create`
- `streamr resend` is now `streamr stream resend`

Reply:
Added the info 👍